### ISSUE TITLE
Escaped quotes test failing

### ISSUE THREAD

**Issue by waylan** [Python-Markdown/markdown#264]
When running the included PHP tests I'm getting the following failure:

```
======================================================================
MarkdownSyntaxError: TestSyntax: "php/PHP-Specific Bugs"
----------------------------------------------------------------------
MarkdownSyntaxError: Output from "/home/pi/md/tests/php/PHP-Specific Bugs.text" failed to match expected output.

--- /home/pi/md/tests/php/PHP-Specific Bugs.xhtml
+++ actual_output.html
@@ -5,13 +5,13 @@
   Headers below should appear exactly as they are typed (no backslash added or removed).
 </p>
 <h1>
-  Header "quoted\" again \""
+  Header "quoted" again \""
 </h1>
 <h2>
-  Header "quoted\" again \""
+  Header "quoted" again \""
 </h2>
 <h3>
-  Header "quoted\" again \""
+  Header "quoted" again \""
 </h3>
 <p>
   Test with tabs for <code>_Detab</code>:


----------------------------------------------------------------------
Ran 292 tests in 54.644s

FAILED (MarkdownSyntaxError=1, SKIP=24)
```

I haven't had a chance to debug it yet,but didn't want to forget about it.

**Comment by waylan** [comment]
Can't seem to replicate this on another system. As this is for a PHP specific bug anyway, I'm closing this and ignoring it for now.

**Comment by waylan** [comment]
This has reappeared. After inplementing Travis CI, this is failing [there](https://travis-ci.org/waylan/Python-Markdown/jobs/21473956). In a comment on #299 @mitya57 stated:

> Re the failing test, I don't see why “Headers below should appear exactly as they are typed (no backslash added or removed)”. I.e., if I put a single backslash, I want to escape the next character, and if I want a real backslash, I would put a double backslash.
> 
> Also, can you reproduce it locally (and if yes, on what OS)? I can't.

I believed I reproduced this on a RaspberryPI, but not anywhere else. Still, the fact that it is giving different results in some situations is concerning. That said, I haven't really worked out what is being tested here (this is a test from PHP's test suite). It may be appropriate to simply skip this test.

I'm reopening thIs issue until a definitive solution can be reached.

**Comment by waylan** [comment]
Fixed this in 85be15973
